sdk/src/main/java/io/dapr/client/domain/SubscribeConfigurationResponse.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.client.domain;
+
+import java.util.List;
+
+/**
+ * Domain object for response from subscribeToConfiguration API.
+ */
+public class SubscribeConfigurationResponse {
+  private final String id;

```

Comment:
add java doc for the fields. Should this field be `subscriptionId` to make it explicit that it is the ID for the subscription and not necessary for individual configuration items\?